src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-wsdlservices.js

```diff
@@ -24,55 +24,72 @@
  * THE SOFTWARE.
  */
 
+let frontPage, mainPage, clientsTab, clientInfo, clientServices, operationDetails, sslCheckFail;
+let addSubjectsPopup, removeAccessRightPopup, removeAllAccessRightsPopup, serviceChangePopup, serviceDetails,servicesPopup ;
+
 module.exports = {
-  tags: ['ss', 'clients', 'wsdlservices'],
+  tags: ['ss', 'clients', 'wsdlservices', 'WIP'],
```

Comment:
Is this tag here intentionally?

Reply:
No, it's to run only that test i am working on, I will remove it.

Reply:
Seems to still be in the PR?